index.html

```diff
@@ -0,0 +1,228 @@
+                        <li>
+                            <img src="images/speaker-2.png" alt="speaker number
+                                one">
+                            <div>
+                                <h3 class="bold speaker-name"></h3>
+                                <p class="orange-color bold speaker-job"></p>
+                                <hr class="speaker-hr">
+                                <p class="speaker-graduated"></p>
+                            </div>
+                        </li>
+                        <li class="hide-on-mobile">
+                            <img src="images/speaker-3.png" alt="speaker number
+                                one">
+                            <div>
+                                <h3 class="bold speaker-name"></h3>
+                                <p class="orange-color bold speaker-job"></p>
+                                <hr class="speaker-hr">
+                                <p class="speaker-graduated"></p>
+                            </div>
+                        </li>
+                        <li class="hide-on-mobile">
+                            <img src="images/speaker-4.png" alt="speaker number
+                                one">
+                            <div>
+                                <h3 class="bold speaker-name"></h3>
+                                <p class="orange-color bold speaker-job"></p>
+                                <hr class="speaker-hr">
+                                <p class="speaker-graduated"></p>
+                            </div>
+                        </li>
+                    </ul>
+                    <div id="more-button-wrapper">
+                        <button>More <i class="bi bi-chevron-down orange-color"></i></button>
+                    </div>
+                </section>
+
+                <section id="partners">
+                    <h1>Partners</h1>
+                    <hr class="short-orange-hr">
+                    <div id="partner-items-wrapper">
+                        <ul>
+                            <li>Google</li>
+                            <li>Facebook</li>
+                            <li>Uber</li>
+                            <li class="last-tow-partners">airbnb</li>
+                            <li class="last-tow-partners">Naver</li>
+                        </ul>
+                    </div>
+                </section>
+
+                <footer>
+                    <h4>Dev Courses 2021</h4>
+                    <p>2021 Dev Courses<br>
+                        All Rights Reserved</p>
+                </footer>
+
+
+                <script
+                    src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/js/bootstrap.bundle.min.js"
+                    integrity="sha384-MrcW6ZMFYlzcLA8Nl+NtUVF0sA7MsXsP1UyJoMp4YLEuNSfAP+JcXn/tWtIaxVXM"
+                    crossorigin="anonymous"></script>
+                <script src="script.js"></script>
+
+
```

Comment:
- [ ] OPTIONAL
Kindly ensure not to add leave empty lines in your code which are not serving any purpose, so as to keep your code cleaner.